packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -878,6 +879,14 @@ class SemanticsData with Diagnosticable {
   ///  * [ScrollPosition.pixels], from where this value is usually taken.
   final double? scrollPosition;
 
+  /// Indicates the total quantity of available content if the node is
+  /// scrollable.
+  ///
+  /// See also:
+  ///
+  ///  * [ScrollPosition.extentTotal], from where this value is usually taken.
+  final double? scrollExtentTotal;
```

Comment:
I think you can get this by scrollExtentMax- scrollExtentMin + rect.height( or width). Do we really need a new property?

Reply:
Thanks for pointing this out, I don't think we need a new parameter then.